packages/mwp-core/src/util/inlineStyle.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+import React from 'react';
+import { getStyles } from 'simple-universal-style-loader';
+
+/**
+ * Returns inline style tags from CSS accumulated by
+ * `simple-universal-style-loader`. This allows us to
+ * render styles from CSS modules on the server.
+ *
+ * @returns {Array} - array of style tags
+ */
+export function getInlineStyleTags(): array {
```

Comment:
Probably needs a corresponding unit test

Reply:
Not sure what Flow says about the `: array` return type but I _think_ it's recommended to use `Array<type>` to specify the type of the elements an array contains, e.g. `Array<React$Element>` or something for this (not 100% confident of the React type, but it should be easy to confirm).

That will also show you that unless you know the return type of `getStyles`, you could be returning a falsy non-array value rather than guaranteeing an Array return type.

Reply:
It _seems_ like the right annotation is `: Array<React.Element<'style'>>`, but typecheck is failing:

```
Cannot get React.Element because property Element is missing in object
type [1].
```

fwiw, the docs only mention a [primitive annotation for function returns](https://flow.org/en/docs/types/functions/#toc-function-returns).

Reply:
I suspect a Flow version thing - I think the syntax might actually be `React$Element<'style'>` or `React$Element`

Reply:
Success. Pushing the updated type annotation; otherwise this looks ready for a final pass.